plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -38,6 +44,13 @@ typedef union _START_RESP {
 	guint32 legacy_resp;
 } START_RESP;
 
+typedef struct _fu_cros_ec_usb_block_info {
```

Comment:
The `_fu_cros_ec_usb_block_info` is redundant, no?

Reply:
Done.